src/server/Elsa.Server.Api/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -54,7 +56,11 @@ public static IServiceCollection AddElsaApiEndpoints(this IServiceCollection ser
                 .AddSingleton<IEndpointContentSerializerSettingsProvider, EndpointContentSerializerSettingsProvider>()
                 .AddAutoMapperProfile<AutoMapperProfile>()
                 .AddSignalR();
-                
+            services.AddMvc(options =>
+            {
+                //use this convertion to set ElsaNewtonsoftJsonConvention to all controllers in Elsa.Server.Api
```

Comment:
Please mind capitalization ("// Use this...").

There's a small typo: "convertion" => "convention").